src/main/webapp/js/puzzle-loader.js

```diff
@@ -7,6 +7,50 @@ const parameterUsername = urlParams.get('user');
 if (!parameterUsername) {
     window.location.replace('/');
 }
+
```

Comment:
delete this comment if you are implementing the correct.html page & things

Reply:
ok, do I need to put that somewhere else?

Reply:
oh nvm I see what you're telling me to delete